ADDRESSING.md

```diff
@@ -75,7 +72,7 @@ Read more: [notes on addressing with HTTP](#notes-on-addressing-with-http).
 
 ### Paths
 
-When site isolation does not matter gateway can expose IPFS namespaces as regular URL paths:
+Outside of browser context, and in cases when site isolation does not matter, a gateway can expose IPFS namespaces as regular URL paths under a single origin:
```

Comment:
add big warning about the danger here? ideally w/ link to explanation in our docs?